src/App.js

```diff
@@ -1,18 +0,0 @@
-import "./App.css";
-import { Footer } from "./components/Home/Footer";
-import { Navbar } from "./components/Home/Navbar";
-import { MainRoutes } from "./routes/MainRoutes";
-function App() {
-  return (
-    <div >
-      <Navbar />
-      <div style = {{minHeight:'500px'}}>
```

Comment:
Never use inline styling. It makes the ability to load the html slower. Always use a seperate css file or make use of styled-components